styles.css

```diff
@@ -0,0 +1,97 @@
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-family: 'Roboto', sans-serif;
+    font-size: 20px;
+}
+
+main {
+    margin: 50px;
+}
+
+.writtenInstructions {
+    background-color: #7ABC31;
+    border: 3px solid #000;
+    border-radius: 10px;
+    padding: 15px;
+}
+
+h2 {
+    text-align: center;
+    font-family: 'Patua One', serif;
+    font-size: 4rem;
+}
+
+
+.instructionMain {
+    display: flex;
+    align-items: center;
+}
+
+.buttons {
+    display: flex;
+    justify-content: space-evenly;
+    width: 100%;
+}
+
+
+.gameNav {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+}
+
+.mainGame {
+    display: flex;
```

Comment:
When these properties are all deselected, there's no change in the browser - is this future coding or can it be removed?

Reply:
I've removed justify content, however, when I remove display: flex and flex-direction: column the layout goes to the left and messes up.